### ISSUE TITLE
Verboseness of verbose flag

### ISSUE THREAD

**Issue by stefanbuck** [RoadieHQ/backstage-entity-validator#19]
## Describe Request:

With the verbose flag set to `true` (default) the output includes a dump of every yaml file the validator is processing. This becomes quite noisy in a monorepo. I wonder what the discussion process was behind adding this output and if you would be open to remove it or put it behind another flag.

**Comment by kissmikijr** [comment]
@stefanbuck So you'd like to see the standard log lines and only skip the yaml dumping at the end, am I right? What do you think printing instead of the file contents only the file paths would be more helpful?

**Comment by stefanbuck** [comment]
Yes, only the standard output and as you suggested the file paths. We run `backstage-entity-validator` on CI with hundreds of files to validated and the output is quite verbose in this case.
